tasks/main.yml

```diff
@@ -43,6 +43,7 @@
 
 - name: Reboot Riak to accept the new config and ensure it is enabled to start at bootup
   service: name=riak enabled=yes state=restarted
+  when: riak_init_system == 'system'
```

Comment:
Ansible is designed to use the default init system for the platform, so thats why I opted for system init system.. if that makes sense?